tox.ini

```diff
@@ -1,10 +1,8 @@
 [tox]
 minversion = 3.14.6
 envlist =
-    py{37,38,39,310}-django32-wagtail{215,216}-factoryboy32
-    py{38,39,310}-django40-wagtail{216,30}-factoryboy32
-    py{38,39,310}-dj40-wagtail40-factoryboy32
-    py{39,310}-dj41-wagtail40-factoryboy32
+    py{37}-django{32}-wagtail{41,42,50}-factoryboy32
+    py{38,39,310,311}-django{32,40,41}-wagtail{41,42}-factoryboy32
```

Comment:
officially Django 3.2 supports up to Python 3.10
https://docs.djangoproject.com/en/3.2/releases/3.2/ but I suppose this is ok